recipes/aaf/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class AafConan(ConanFile):
+    name = "aaf"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://sourceforge.net/projects/aaf/"
+    description = "A  cross-platform SDK for AAF. AAF is a metadata management system and file format for use in professional multimedia creation and authoring."
+    topics = ("aaf", "multimedia", "crossplatform")
+    license = "AAF SDK PUBLIC SOURCE LICENSE"
```

Comment:
```suggestion
    license = "AAFSDKPSL-2.0"
```

There is no official short form, but I would suggest following the license file name + version, like GPL-2.0, BSL-1.0, Apache-2.0 and so on ...